### DISCUSSION TITLE
How to intall TeslaMate on Hassio OS

### DISCUSSION THREAD

**Discussion by nutsofamily** [teslamate-org/teslamate#3118] [Getting started]
As there is no docker install how do I install on Pi4 running Hassio OS. Is there a custom repository I can add? Tries the GitHub url but that wont work.

**Comment by KennethS2022** [comment]
Have you tried this?  I have not tried it myself but saw it and thought it may be helpful for you.

[https://github.com/matt-FFFFFF/hassio-addon-teslamate](url)

**Reply by jb3112** [reply]
ANy where to get help, i have tried but havent got it to work, im very beginner at HA
